src/apps/competitions/tasks.py

```diff
@@ -247,7 +230,7 @@ def unpack_competition(competition_dataset_pk):
 
     # Children datasets are those that are created specifically for this "parent" competition.
     # They will be deleted if the competition creation fails
-    children_datasets = []
+    # children_datasets = []
```

Comment:
pull?

Reply:
I was leaving it as a placeholder/reminder to eventually actually implement that. I can pull it, or mark it with a TODO, or whatever makes the most sense.

Reply:
`# TODO: ` would be fine